src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -218,6 +205,35 @@ class SidebarPanels extends React.Component<Props> {
                         )}
                     />
                 )}
+                {/* This handles both the default activity sidebar and the activity sidebar with a
+                comment or task deeplink.  */}
+                {hasActivity && (
+                    <Route
+                        exact
+                        path={[
+                            `/${SIDEBAR_VIEW_ACTIVITY}`,
+                            `/${SIDEBAR_VIEW_ACTIVITY}/:activeFeedItemType/:activeFeedItemId`,
+                        ]}
+                        render={({ match }) => {
+                            const activeFeedItemType = match.params.activeFeedItemType
```

Comment:
"Item" usually refers to a file or folder. Can the variable names be more generic, such as `activityId` and `activityType`?